app/api/metrics/route.ts

```
@@ -0,0 +1,60 @@
+	// Allow unrestricted access in development
+	return true;
+}
+
+// Alternative authenticated endpoint
+export async function POST(request: NextRequest) {
+	if (!isAuthorized(request)) {
+		return new NextResponse("Unauthorized", { status: 401 });
+	}
+
+	return GET(request);
+}
```

Biome (2.1.2)

[error] This async function lacks an await expression.

Remove this async modifier, or add an await expression in the function.

Async functions without await expressions may not need to be declared async.

(lint/suspicious/useAwait)